### PR TITLE
feat: add presets for different transactions

### PR DESCRIPTION
**Pull request by dzbo**
Allow to select presets for different types of transactions 

CleanShot 2022-11-17 at 11 53 22@2x

### CODE REVIEW

**Comment by frozeman** [comment]
Task linked: [CU-32bfj81 [Token Tx] ERC20 Token Transfer](https://app.clickup.com/t/32bfj81)

File: src/components/endpoints/SendLyx.vue [hunk 1]
```diff
@@ -8,17 +8,99 @@ import { TransactionConfig } from 'web3-core'
 import useWeb3 from '@/compositions/useWeb3'
 import { DEFAULT_GAS, DEFAULT_GAS_PRICE } from '@/helpers/config'
 
+type TransactionSelect = {
+  label: string
```
Comment (Hugoo) [inline src/components/endpoints/SendLyx.vue:12]: the lint says this is never used ?

**Review by Hugoo — APPROVED** [review]
cool ! let's add emojis to make it cooler 😎 :p

**Comment by dzbo** [comment]
> cool ! let's add emojis to make it cooler 😎 :p

haha was thinking about this but couldn't found anything good for the tokens ;) let me know if you have any suggestions

**Pull request merged**